src/content/docs/en/guides/integrations-guide/vercel.mdx

```diff
@@ -427,56 +395,6 @@ export default defineConfig({
 });
 ```
 
-Optionally, you can create a file recognized by the adapter named `vercel-edge-middleware.(js|ts)` in the [`srcDir`](/en/reference/configuration-reference/#srcdir) folder to create [`Astro.locals`](/en/reference/api-reference/#astrolocals).
-
-Typings requires the [`@vercel/edge`](https://www.npmjs.com/package/@vercel/edge) package.
-
-```js title="src/vercel-edge-middleware.js"
-/**
- *
- * @param options.request {Request}
- * @param options.context {import("@vercel/edge").RequestContext}
- * @returns {object}
- */
-export default function ({ request, context }) {
-  // do something with request and context
-  return {
-    title: "Spider-man's blog",
-  };
-}
-```
-
-If you use TypeScript, you can type the function as follows:
-
-```ts title="src/vercel-edge-middleware.ts"
-import type { RequestContext } from '@vercel/edge';
-
-export default function ({ request, context }: { request: Request; context: RequestContext }) {
-  // do something with request and context
-  return {
-    title: "Spider-man's blog",
-  };
-}
-```
-
-The data returned by this function will be passed to Astro middleware.
-
-The function:
-
-* must export a **default** function;
-* must **return** an `object`;
-* accepts an object with a `request` and `context` as properties;
-* `request` is typed as [`Request`](https://developer.mozilla.org/en-US/docs/Web/API/Request);
-* `context` is typed as [`RequestContext`](https://vercel.com/docs/concepts/functions/edge-functions/vercel-edge-package#requestcontext);
-
-#### Limitations and constraints
-
-When you opt in to this feature, there are few constraints to note:
-
-* The Vercel Edge middleware will always be the **first** function to receive the `Request` and the last function to receive `Response`. This an architectural constraint that follows the [boundaries set by Vercel](https://vercel.com/docs/concepts/functions/edge-middleware).
-* Only `request` and `context` may be used to produce an `Astro.locals` object. Operations like redirects, etc. should be delegated to Astro middleware.
-* `Astro.locals` **must be serializable**. Failing to do so will result in a **runtime error**. This means that you **cannot** store complex types like `Map`, `function`, `Set`, etc.
```

Comment:
This is the one bit of information that remains relevant - the locals added by the user will need to be JSON serializable. I think we communicate this adequately in the section above.